UI/PaymentBoosterConfigProvider.php

```diff
@@ -1,7 +1,7 @@
 <?php
 declare(strict_types=1);
 
-namespace Bold\CheckoutPaymentBooster\UI\Payment;
+namespace Bold\CheckoutPaymentBooster\UI;
```

Comment:
Moved the config provider up into a more global scope so we can access it outside of the payment step. Open to suggestions on handling this a different way.